### PR TITLE
Start stop encoders

### PR DESCRIPTION
**Pull request by hugomatic**
This adds logic to start / stop video encoders (2 for now).

The first message to start is sent when the user toggles the switch (setting a "starting" status). During the daemon report callback, if the encoders are not all started, then more messages are sent. 

In this PR, the UI is not "aware" of the "starting" / "stopping" status. A single "stopped, starting, started, stopping" list could replace the 4 booleans value.  One thing to note is that if the video is "starting" and the user presses the switch, the backend will go to "stopping" mode. This behavior might surprise the user.

Next, the same recipe should be applicable to audio and "Pull over" (using the planner msg instead of the daemon report). These should be simpler since there's only one audio encoding node, and one pad planner message to send.

### CODE REVIEW

File: modules/dreamview/backend/teleop/teleop.cc [hunk 1]
```diff
@@ -182,37 +216,94 @@ void TeleopService::UpdateModem(const std::string &modem_id,
...
+      teleop_status_["mic"] = voipIsRunning;
+  }
+}
+
+void TeleopService::SendVideoStreamCmd(bool start_stop) {
+  DaemonServiceCmd msg;
+  if (start_stop) {
+    msg.set_cmd("start");
+  }
+  else {
```
Comment (xiaoxq) [inline modules/dreamview/backend/teleop/teleop.cc]: Please fix the style issue with `scripts/clang-format.sh`.

File: modules/dreamview/backend/teleop/teleop.cc [hunk 2]
```diff
@@ -182,37 +216,94 @@ void TeleopService::UpdateModem(const std::string &modem_id,
...
+          if (service.find("voip_encoder") != std::string::npos) {
+              voipIsRunning = true;
+              break;
+          }
       }
-    }
-    boost::unique_lock<boost::shared_mutex> writer_lock(mutex_);
-    teleop_status_["mic"] = voipIsRunning;
+      boost::unique_lock<boost::shared_mutex> writer_lock(mutex_);
+      teleop_status_["mic"] = voipIsRunning;
```
Comment (xiaoxq) [inline modules/dreamview/backend/teleop/teleop.cc]: Wrong indents.

File: modules/dreamview/backend/teleop/teleop.cc [hunk 3]
```diff
@@ -18,6 +18,8 @@
 
 #include "cyber/common/log.h"
 #include "google/protobuf/util/json_util.h"
+#include "cyber/time/time.h"
+#include "modules/common/util/message_util.h"
```
Comment (xiaoxq) [inline modules/dreamview/backend/teleop/teleop.cc]: Make the headers well sorted.

File: modules/dreamview/backend/teleop/teleop.cc (hunk 1 above)
Comment (hugomatic) [reply modules/dreamview/backend/teleop/teleop.cc]: fixed [here](https://github.com/ApolloAuto/apollo/pull/10106/commits/e52de49fccb6d559e548bf24fda32d77c7930df3)

**Pull request merged**